lib/symbValue.ml

```diff
@@ -384,6 +390,10 @@ module Make(Cst:Constant.S) = struct
 (* Assume concrete and others always to differ *)
   | (Val (Symbolic _|Label _|Tag _|ConcreteVector _|PteVal _), Val (Concrete _))
   | (Val (Concrete _), Val (Symbolic _|Label _|Tag _|ConcreteVector _|PteVal _)) -> zero
+  | (Val Instruction(i1), Val Instruction(i2)) ->
+    if InstrLit.compare i1 i2 == 0
```

Comment:
Use `InstrLit.eq`, if this function is added.

Reply:
Coming in the next commit.